src/args.rs

```diff
@@ -157,6 +157,16 @@ impl Arguments {
     pub fn from_args() -> Self {
         structopt::StructOpt::from_args()
     }
+
+    /// Like from_args(), but operates on an explicit iterator and not the global arguments.
```

Comment:
Please use monospace here:

```suggestion
    /// Like `from_args()`, but operates on an explicit iterator and not the global arguments.
```

Reply:
Thanks, updated!